Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -576,6 +576,19 @@ final class StringExtensionsTests: XCTestCase {
         XCTAssertFalse("notanemail.com".matches(pattern: "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}"))
         XCTAssertTrue("email@mail.com".matches(pattern: "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}"))
     }
+    
+    func testRegexMatchOperator() {
+        #if canImport(Foundation)
```

Comment:
Sorry to be a pain, but can you move the `#if` to outside the function?

Reply:
Sure, no problem